addon/components/detail.js

```diff
@@ -242,7 +246,9 @@ export default Component.extend(SpreadMixin, HookMixin, PropTypeMixin, {
   applyStoreUpdate ({lastAction, newProps, validationResult, value}) {
     if (Object.keys(newProps).length !== 0) {
       const model = newProps.renderModel || this.get('renderModel')
-      const view = newProps.view ? this.getRenderView(model, newProps.view) : this.get('renderView')
+      const view = newProps.view
+        ? this.getRenderView(model, newProps.view)
+        : this.getRenderView(model, this.get('view'))
```

Comment:
Yeah, I was having trouble getting the new normalized view from the computed property in the cases where I needed it. I think this can all be smoothed out with some slight restructuring of this component.